### PR TITLE
Feature: get group members pluggable

### PR DESCRIPTION
**Pull request by jensens**
- see #39 
- needs review
- contains already branch `testfixes`

### CODE REVIEW

**Comment by agitator** [comment]
@tisto ping?

**Comment by agitator** [comment]
@tisto ping?

**Comment by tisto** [comment]
@agitator sorry for not replying! I need to make a 1.0 release and talk to my customer about how to proceed with the plugin. I'm happy to review/merge the pr afterwards. I will make a release over the weekend, talk to the customer on Monday and then get back to you, ok?

**Comment by agitator** [comment]
thanx!

File: src/pas/plugins/memberpropertytogroup/interfaces.py [hunk 1]
```diff
@@ -364,3 +364,20 @@ def validate_filled_field_couples(data):
...
 class IMPTGPlugin(Interface):
     """Member Properties To Group Plugin"""
+
+
+class IGetGroupMembers(Interface):
+    """List group members from a MPTG plugin
+
+    This is use case specific.
+    By default no group members are listed.
+    In some cases its possible to liste members.
```
Comment (tisto) [inline src/pas/plugins/memberpropertytogroup/interfaces.py]: @jensens typo. :)

**Pull request merged**